src/main/java/book/store/service/impl/UserServiceImpl.java

```diff
@@ -33,7 +33,11 @@ public UserRegistrationResponseDto register(UserRegistrationRequestDto requestDt
         }
         User user = userRegistrationMapper.toModel(requestDto);
         user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
-        user.setRoles(Set.of(roleService.findRoleByName(Role.RoleName.USER)));
+        if (user.getEmail().startsWith("admin")) {
```

Comment:
It's better to create a separate endpoint that will create admins than to hard code around like this